.github/workflows/i18n-string-extract-master.yml

```diff
@@ -0,0 +1,36 @@
+name: Extract and upload i18n strings
+
+on:
+  push:
+    branches:
+      - master
+  workflow_dispatch:
+
+jobs:
+  extract-and-upload-i18n-strings:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v4
+    - name: Set up environment
+      uses: ./.github/actions/setup
+    - name: Configure i18n client
+      run: |
+        pip install wlc
+    - name: Generate i18n strings
+      run: yarn generate:i18n
+    - name: Upload i18n strings
+      run: |
+        if [[ ! -f packages/desktop-client/locale/en.json ]]; then
+          echo "File packages/desktop-client/locale/en.json not found. Ensure the file was generated correctly."
+          exit 1
+        fi
```

Comment:
Could this be a seprate step?

Reply:
You mean like having a step just for the validation? My thinking is this was just a simple prereq check for the `wlc` command below, but open to refactoring if you think they should be separate